### ISSUE TITLE
is there a way to use  --build-arg during image build

### ISSUE THREAD

**Issue by ehaselwanter** [coreos/quay-docs#62]
I do not find anything about build-args in the docs. I guess they are not supported. 
If they are not supported, would you mind pointing people into the right direction?

Use case: be able to support build date label:

```
BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` .
```

and in the Dockerfile:

```
LABEL org.label-schema.build-date=$BUILD_DATE
```

with

```
docker build --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` .
```

**Comment by josephschorr** [comment]
Hi @ehaselwanter: There is not, currently, but we are working on a solution that will allow them to be specified.

**Comment by blueyed** [comment]
@josephschorr 
There is nothing like that yet, is there?

**Comment by josephschorr** [comment]
@blueyed: Nope. Still on the roadmap.

**Comment by chrislovecnm** [comment]
When?  Can we re-open this?

I would like to be able to run a build arg off of a github webhook.  To be clear about the requirements.

**Comment by tomwans** [comment]
Is this still in the mix?

**Comment by Antiarchitect** [comment]
Trying to find any SaaS that supports `--build-arg` feature.

**Comment by foutoucour** [comment]
@Antiarchitect hey did you find any? See Dockerhub has the same issue :(

@josephschorr any other issue we can follow to have a view on the progress on the case?

**Comment by managerger** [comment]
Do we have the way to specify ARGS for Dockerfile while triggering building? Why the thread was closed?

**Comment by webknjaz** [comment]
@josephschorr could you please reopen this?

**Comment by josephschorr** [comment]
@webknjaz Unfortunately no, since this isn't the place for Quay feature requests. We're already tracking this internally on our roadmap.

**Comment by webknjaz** [comment]
@josephschorr anywhere public?

**Comment by josephschorr** [comment]
@webknjaz Not currently, no. We'll be changing that soon, but I can't speak more to it at this time.

**Comment by foutoucour** [comment]
Hello @josephschorr 
Do we have an ETA on this feature? Most of our docker images use build args. It prevents us to use your product at full power.

**Comment by theRemix** [comment]
As a workaround, I've been using docker entrypoint to read runtime envs to configure the container at runtime.

still waiting for Quay to enable `--build-arg`

**Comment by blackandred** [comment]
It would be good to have a possibility to label images with build date, git tag, git commit id.

**Comment by AlanCoding** [comment]
As an experiment, I wanted to see what happened if I added a build arg without a default:

https://quay.io/repository/alancoding/awx-ee/build/23ecfdf7-9642-4587-9ee5-e7807e31f480

Unsurprisingly, it doesn't provide a value, so the build fails.

I'm trying to think if there's any workaround on the build level - like split out my 1 Dockerfile into 3, where 1 has all the logic and the other 2 just put in different values for the parameter. Even if I had to create multiple projects in quay, that's would be fine. I have not figured out any way to do this.
